node/cli/src/service.rs

```diff
@@ -206,13 +208,24 @@ pub fn start_service<S>(
         adapters::signed_transaction_to_payload::spawn_task(transactions_rx, payload_tx.clone());
         adapters::receipt_transaction_to_payload::spawn_task(receipts_rx, payload_tx.clone());
 
-        spawn_consensus_task_fn(
-            inc_gossip_rx,
-            payload_rx,
-            out_gossip_tx,
-            consensus_control_rx,
-            beacon_block_consensus_body_tx,
-        );
+        if let Some(devnet_cfg) = devnet_cfg {
+            spawn_consensus(
+                inc_gossip_rx,
+                payload_rx,
+                out_gossip_tx,
+                consensus_control_rx,
+                beacon_block_consensus_body_tx,
+                devnet_cfg.block_period,
+            );
+        } else {
+            spawn_task(
```

Comment:
Prefer `passthrough::spawn_consensus` and `txflow_task::spawn_task` for better readability.

Reply:
Fixed in 4db1cfe